chef/data_bags/crowbar/template-provisioner.schema

```diff
@@ -19,6 +19,7 @@
             "access_keys": { "type": "str", "required": true },
             "shell_prompt": { "type": "str", "required": true },
             "enable_pxe": { "type": "bool", "required": true },
+            "self_update_url": { "type": "str", "required": true },
```

Comment:
Sorry, I failed to notice this earlier on, but we have a `suse` attribute, and this should live in there. See line 56.